model/lib/email/email_content.dart

```diff
@@ -18,8 +18,6 @@ extension EmailContentExtension on EmailContent {
   String get asHtml {
     if (type == EmailContentType.textPlain) {
       return content
-          .replaceAll("'", r"\'")
-          .replaceAll('"', r'\"')
```

Comment:
I am worried this may have side effect :(

Reply:
IMO, It won't be affected whatsoever. WebView has a good handling mechanism to recognize these characters.